pkg/scheduler/framework/runtime/framework.go

```diff
@@ -605,8 +607,9 @@ func (f *frameworkImpl) runPostFilterPlugin(ctx context.Context, pl framework.Po
 // and add the nominated pods. Removal of the victims is done by
 // SelectVictimsOnNode(). Preempt removes victims from PreFilter state and
 // NodeInfo before calling this function.
-func (f *frameworkImpl) RunFilterPluginsWithNominatedPods(ctx context.Context, state *framework.CycleState, pod *v1.Pod, info *framework.NodeInfo) *framework.Status {
+func (f *frameworkImpl) RunFilterPluginsWithNominatedPods(ctx context.Context, state *framework.CycleState, pod *v1.Pod, info *framework.NodeInfo) (*framework.Status, sets.String) {
```

Comment:
I don't expect that the plugin that fails in preemption will impact the re-queueing decision, so returning the set of failed plugins isn't necessary.

Reply:
True, PostFilter don't need to return this. Will update.

Reply:
Oh, RunFilterPluginsWithNominatedPods is both used in regular filtering and preemption cases, so it's still necessary to return the failed plugins.

![image](https://user-images.githubusercontent.com/1425903/105286830-e8c84780-5b6b-11eb-8924-aedcb5c04c4c.png)

Regarding the preemption case, the original FitError that embeds the failed plugins will be used, not the ones returned in the preemption path.